src/table/components/head/HeadCellMenu.tsx

```diff
@@ -165,15 +165,20 @@ const HeadCellMenu = ({ column, tabIndex }: HeadCellMenuProps) => {
     if (!openMenuDropdown) resetSelectionActionsEnabledStatus();
   }, [openMenuDropdown, resetSelectionActionsEnabledStatus]);
 
+  const chartId = layout.qInfo.qId as string;
+  const buttonId = `sn-table-head-menu-button-${pageColIdx}-${chartId}`;
+  const menuId = `sn-table-head-menu-${pageColIdx}-${chartId}`;
```

Comment:
I was thinking about making `sn-table-head-menu-` and the other one into constants, but didn't really have a good place to store them, without creating a circular dep